src/main/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoader.java

```diff
@@ -59,6 +61,10 @@ protected void init() {
                     activeVenueDetailsMapByPostcode.put(line[8], venueDetails);
                     activeVenueDetailsMapByEpimsId.put(line[15], venueDetails);
                 }
+                venueDetailsMapByRpc.put(RegionalProcessingCenter.builder()
```

Comment:
Will this work correctly? Does the equals figure out the same RPC is being added?